payload/import/mkwNet.hpp

```diff
@@ -7,6 +7,10 @@ namespace mkw::Net
 
 #if RMC
 
+// Allow clients to modify their region without having to change their matching
+// area.
+#  define CUSTOM_MATCHING_AREA_ADDRESS 0x80005EFC
```

Comment:
This should be defined as a symbol (using the AT macro) rather than using a hardcoded address, so that the loader can relocate it if needed

Reply:
Addressed in commit [9f9445c](https://github.com/WiiLink24/wfc-patcher-wii/pull/40/commits/9f9445cf3a1ba2ffc751fab1ef37e2d277a3d91d).